internal/cluster/validator.go

```diff
@@ -600,3 +601,16 @@ func (v *clusterValidator) skipNetworkHostPrefixCheck(c *clusterPreprocessContex
 	}
 	return false
 }
+
+// getHostPrefix returns the default host prefix to use.
+func getHostPrefix(skipHostPrefix bool, clusterNetwork *models.ClusterNetwork) int {
+	hostPrefix := int(clusterNetwork.HostPrefix)
+	if skipHostPrefix {
+		if network.IsIPv6CIDR(string(clusterNetwork.Cidr)) {
+			hostPrefix = DefaultIPV6HostPrefix
```

Comment:
If the host prefix needs to be skipped, why does it have a value?
If it has a value then it is not skipped.

Reply:
The name of this function is confusing and I'll change it. The prefix is only needed for CIDR validation, if the prefix check is skipped we still need to use a default value for CIDR validation check.